### PR TITLE
CAI Adjustments

### PR DESCRIPTION
**Pull request by Mach565**
Removed some unused units from one file, and in addition CAI has some more build options.

### CODE REVIEW

**Review by sprunk — APPROVED** [review]

**Comment by Mach565** [comment]
Been trying to test it out, but CAI isn't doing anything, no idea why.

**Review by sprunk — CHANGES_REQUESTED** [review]
i'll check it in about 24 hours

File: LuaRules/Configs/cai/buildtasks.lua [hunk 1]
```diff
@@ -499,9 +504,9 @@ factionBuildConfig = {
 				
 				[7] = { -- riot
 					importanceMult = 0.6,
-					count = 2,
-					[2] = {ID = UnitDefNames['amphimpulse'].id, chance = 0.7},
-					[1] = {ID = UnitDefNames['amphriot'].id, chance = 0.3},
+					count = 3,
```
Comment (sprunk) [inline LuaRules/Configs/cai/buildtasks.lua:507]: ?

Comment (Mach565) [reply LuaRules/Configs/cai/buildtasks.lua:507]: forgot to change it back, my bad

**Comment by Mach565** [comment]
Fixed the issue. Non-bombers in the bombers category breaks the AI and stops it from doing anything.

**Review by sprunk — APPROVED** [review]
Looks okay but let GF have a look too.

**Pull request merged**